src/Bicep.LangServer/Handlers/BicepEditLinterRuleCommandHandler.cs

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Bicep.Core;
+using Bicep.Core.Configuration;
+using Bicep.LanguageServer.Configuration;
+using Bicep.LanguageServer.Telemetry;
+using MediatR;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using OmniSharp.Extensions.JsonRpc;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Extensions.LanguageServer.Protocol.Server;
+using OmniSharp.Extensions.LanguageServer.Protocol.Window;
+using OmniSharp.Extensions.LanguageServer.Protocol.Workspace;
+using Range = OmniSharp.Extensions.LanguageServer.Protocol.Models.Range;
+
+namespace Bicep.LanguageServer.Handlers
+{
+    /// <summary>
+    /// Handles the internal command for code actions to edit a particular linter rule in the bicepconfig.json file
+    /// </summary>
+    public class BicepEditLinterRuleCommandHandler : ExecuteTypedCommandHandlerBase<DocumentUri, string, string>
+    {
+        private readonly string DefaultBicepConfig;
+        private readonly ILanguageServerFacade server;
+        private readonly ITelemetryProvider telemetryProvider;
+
+        public BicepEditLinterRuleCommandHandler(ISerializer serializer, ILanguageServerFacade server, ITelemetryProvider telemetryProvider)
+            : base(LanguageConstants.EditLinterRuleCommandName, serializer)
+        {
+            DefaultBicepConfig = DefaultBicepConfigHelper.GetDefaultBicepConfig();
+            this.server = server;
+            this.telemetryProvider = telemetryProvider;
```

Comment:
Looks like we aren't logging telemetry anywhere. Do we plan to capture any telemetry data here? If not, can we remove it?

Reply:
removed for now 
8d017dbbc4be4dd462e831ca30b845528d7c4fb2